.github/workflows/canary-integration-test.yml

```diff
@@ -1468,3 +1468,50 @@ jobs:
         uses: ./.github/workflows/collect-logs
         with:
           name: csi-hostnetwork-disabled
+
+  csi-addons-testing:
```

Comment:
Instead of creating a new test, how about adding it to an existing canary test? The reason we have so many different canary tests is mainly because of different OSD configurations required. But in this case we don't need an entirely new cluster (such as different OSDs), so it keeps the CI simpler with fewer tests if we can just test this in an existing test.

Reply:
yea, this will be better.
I have moved the test under canary test.